src/pages/News.jsx

```diff
@@ -1,11 +1,134 @@
-import Layout from "../components/Layout/Layout";
+import styled from "@emotion/styled"
+import { Stack, Typography } from "@mui/material"
+import CampaignOutlinedIcon from "@mui/icons-material/CampaignOutlined"
+import Accordion from "@mui/material/Accordion"
+import AccordionSummary from "@mui/material/AccordionSummary"
+import AccordionDetails from "@mui/material/AccordionDetails"
+import IconButton from "@mui/material/IconButton"
+import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos"
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos"
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown"
+// import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import React, { useEffect } from "react"
 
+import Layout from "../components/Layout/Layout"
+import Title from "../components/shared/Title"
+import { NewsData } from "../data/NewsData"
 const News = () => {
+  // 연도와 월 데이터 추출하여 중복 제거
+  const years = [...new Set(NewsData.map(item => item.year))]
+  const months = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"]
+
+  const [selectedYear, setSelectedYear] = React.useState(years[0])
+  const [selectedMonth, setSelectedMonth] = React.useState(months[0])
```

Comment:
```suggestion
  const [selectedYear, setSelectedYear] = React.useState(years[0])
  const [selectedMonth, setSelectedMonth] = React.useState(months[0])
```
상태로 관리하는 부분 잘생각했음! 다만 여기 초기값은 현재(오늘날짜) 월과 연이 되어야 할것 같음.